daemon/cmd/ipam.go

```diff
@@ -238,7 +238,7 @@ func (d *Daemon) allocateHealthIPs() error {
 			// In ENI mode, we require the gateway, CIDRs, and the ENI MAC addr
 			// in order to set up rules and routes on the local node to direct
 			// endpoint traffic out of the ENIs.
```

Comment:
nit: Please update this commit to also reflect that the same is true for AlibabaCloud

Reply:
ok, i will change